### ISSUE TITLE
customClass - need to use !important

### ISSUE THREAD

**Issue by jczerosb** [sweetalert2/sweetalert2#1525]
customClass looks a nice resource, but due to CSS order of precedence, we have to use "!important" to get my own style or property applied.

So, would be good to have a note in documentation page alerting that due to precedence conflicting properties between classes can be resolver with "!important"

The use of !important is necessary in some properties not all

**Comment by limonte** [comment]
Good day @jczerosb 

The new patch release (v8.8.2) has been published, please update the `sweetalert2` dependency to the latest and try your use-case, it should work without `!important` now.

Let me know about the result, thank you.

**Comment by limonte** [comment]
The general recommendation is to increase CSS specificity:

```js
customClass: {
  popup: 'my-popup-class',
  confirmButton:  'my-confirm-button-class'
}
```

```css
/* .my-popup-class will improve CSS specificity and increase styles proirity */
.my-popup-class .my-confirm-button-class {
   ...
}
```

**Comment by jczerosb** [comment]
@limonte 
Tested using same sample sent, looks fine, except for "container".
Just in case customClasse listed in my sample are those I detected in current work.

> Read more about CSS specificity: https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity
Nice doc, as far it is logical, in many years is first time I heard about word "Specifity".
Would be nice a note in customClass documentation area, letting anybody to follow "Specificity recomendations"
